examples/bno055_i2c-gpio_simpletest.py

```diff
@@ -0,0 +1,30 @@
+"""
+This example demonstrates how to instantiate the
+Adafruit BNO055 Sensor using this library and just
+the I2C bus number.
+This example will only work on a Raspberry Pi
+and does require the i2c-gpio kernel module to be
+installed and enabled. Most Raspberry Pis will
+already have it installed.
```

Comment:
Add additional commentary here mentioning need to alter config.txt to actually enable the overlay:
```
To enable, modify config.txt...add `dtoverlay=i2c_gpio` etc...
```